Makefile.PL

```diff
@@ -53,13 +53,6 @@ WriteMakefile(
         'Test::MockModule' => '0.170',
 
         'Promise::ES6' => 0.23,
-
-        $ENV{'AUTOMATED_TESTING'} ? (
```

Comment:
Why did these get removed?

Reply:
Because they don’t actually work to avoid the dependency on the CPAN smokers.